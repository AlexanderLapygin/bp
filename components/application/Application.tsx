import * as React from 'react'
import {TasteCourt} from "./taste-court";
import {ApplicationBar} from "./application-bar/ApplicationBar";
import {Box} from "@material-ui/core";
import {makeStyles} from "@material-ui/styles";


const useStyles = makeStyles(() => ({
  root: {
  }
}));

export const Application = () => {
  const classes = useStyles({});
  return (
    <Box className={classes.root}>
      <ApplicationBar title="Taste Club"/>
      <TasteCourt/>
    </Box>
  )
}
