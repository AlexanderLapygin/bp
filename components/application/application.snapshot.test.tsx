import * as React from "react";
import * as TestRenderer from 'react-test-renderer';
import Application from "./Application";

it('renders correctly', () => {
  const tree = TestRenderer.create(<Application/>).toJSON()
  expect(tree).toMatchSnapshot()
})
