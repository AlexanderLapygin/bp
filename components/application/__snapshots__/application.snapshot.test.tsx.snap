// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiBox-root MuiBox-root-4 makeStyles-root-1"
>
  <header
    className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionStatic makeStyles-back-7 MuiAppBar-colorPrimary"
  >
     
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <button
        aria-label="menu"
        className="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-5 MuiIconButton-colorInherit MuiIconButton-edgeStart"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiIconButton-label"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
            />
          </svg>
        </span>
      </button>
      <h5
        className="MuiTypography-root makeStyles-title-6 MuiTypography-h5"
      >
        Taste Club
      </h5>
    </div>
  </header>
  <div
    className="MuiBox-root MuiBox-root-99"
    style={
      Object {
        "height": "100vh",
        "width": "100vw",
      }
    }
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-10 MuiGrid-justify-xs-center"
    >
      <div
        className="MuiGrid-root MuiGrid-item"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
        >
          <div
            className="MuiCardHeader-root"
          >
            <div
              className="MuiCardHeader-content"
            >
              <span
                className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
              >
                First title
              </span>
            </div>
          </div>
          <div
            className="MuiCardContent-root"
          >
            First content
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
        >
          <div
            className="MuiCardHeader-root"
          >
            <div
              className="MuiCardHeader-content"
            >
              <span
                className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
              >
                Second title
              </span>
            </div>
          </div>
          <div
            className="MuiCardContent-root"
          >
            Second content
          </div>
        </div>
      </div>
    </div>
    <header
      className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionRelative makeStyles-background-211 MuiAppBar-colorPrimary"
    >
       
      <div
        className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <h6
          className="MuiTypography-root MuiTypography-h6"
        >
          Playbar
        </h6>
      </div>
    </header>
  </div>
</div>
`;
